packages/firebase_auth/example/test/firebase_auth.dart

```diff
@@ -46,6 +50,37 @@ void main() {
           user2.metadata.lastSignInTime, equals(user2.metadata.creationTime));
     });
 
+    test('email auth', () async {
+      final Random random = Random();
+      final String testEmail = 'testuser${random.nextInt(100000000)}@gmail.com';
```

Comment:
Is there no concern of a clash here which may make the test flaky?

Reply:
It's unlikely since the user is deleted at the end of the test.

I am reluctant to roll my own random string generator since there's already a package for that: https://pub.dev/packages/random_string

Switched to using uuid for now.